configs/components/rubygem-logging.rb

```diff
@@ -1,6 +1,6 @@
 component 'rubygem-logging' do |pkg, settings, platform|
-  pkg.version '2.2.2'
-  pkg.md5sum '13c12082ef0b69a639ed7ca318e7f514'
+  pkg.version '2.3.0'
```

Comment:
I think this release included a fix for this. We will want to *not* patch anymore https://github.com/puppetlabs/puppet-runtime/tree/master/resources/patches/rubygem-logging